src/workbench/notification/statusBarView/index.tsx

```diff
@@ -28,7 +27,9 @@ export function NotificationStatusBarView(
     const renderIcon = hasNotifications ? 'bell-dot' : 'bell';
 
     useEffect(() => {
-        const container = document.getElementById(ID_APP);
+        const container = document.querySelector(
```

Comment:
```suggestion
        const container = $(
```
可以使用 common 中的选择器 API